ResourceManager/Xliff/TranslationUtil.cs

```diff
@@ -248,6 +248,7 @@ private static bool IsTranslatableItemInBox(PropertyInfo property, object itemOb
 
         private static readonly HashSet<string> _translatableItemInComponentNames = new HashSet<string>(StringComparer.Ordinal)
         {
+            "AccessibleName",
```

Comment:
We should add [`AccessibleDescription `](https://docs.microsoft.com/dotnet/api/system.windows.forms.control.accessibledescription) as well

Reply:
added and rebased